src/openrct2/rct2/Limits.h

```diff
@@ -18,6 +18,7 @@ namespace RCT2::Limits
     constexpr const uint8_t MaxTrainsPerRide = 32;
     constexpr const uint8_t DowntimeHistorySize = 8;
     constexpr const uint16_t MaxEntities = 10000;
+    constexpr const uint16_t MaxEntitiesFlag15 = 15000;
```

Comment:
Wouldn't it be better to just make it say MaxEntitiesRCTC

Reply:
Only a small subset of RCTC-exported SV6 saves have this, so that wouldn’t be accurate and also a bit confusing.

Reply:
Can you at least name it something useful. Flag15 means nothing. MaxEntotiesRCTCExtended would mean a lot more